### ISSUE TITLE
JSON:API contentType application/vnd.api+json

### ISSUE THREAD

**Issue by BenLune** [scullyio/scully#1344]
## 🧩 Feature request
Authorize contentType application/vnd.api+json in httpGetJson function for JSON:API format calls

### Description
Hi there,
I'm working on Drupal headless projects with Angular. I'm enthusiastic to use Scully to pre-render the contents.
To get data from Drupal, we use JSON:API. When I use the Scully json plug-in I have an error, but too generic : Could not fetch data for route "/products/:productId".
After couple of hours wondering why it didn't work and trying to understand the reason why it didn't work, I copy paste the the source code of httpGetJson and jsonRoutePlugin. Then I could understand and log the error, it was a contentType error.

### Describe the solution you'd like

Adding application/vnd.api+json to the contentTypes checks to authorize JSON:API calls.

### Describe alternatives you've considered

Overriding the default code to make it work...

**Comment by SanderElias** [comment]
@BenLune We had some difficulties before with API's not returning valid JSON, that's why the strict chack is there.
But you have a valid concern, and we will add some code to cater to additional content types.

**Comment by BenLune** [comment]
Thank you @SanderElias for you answer, I understand, and thank you for the great job :-)
For your information I extended the httpGetJsonApi function to be called as long as there are some contents to load. With JSON:API we have a links object in the answer containing a 'next' variable with the url to be called to get the next contents.
I also have a project where the urls used by the application are coming from Drupal, and managed by the contributors. I created a CustomUrlSerializer to make it work with the Angular Router. I also rewrite the route.map function in JsonPlugIn to get the url from Drupal rather than the product/:productId for example. Everything seems to works fine until now.
